.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["latest"]
```

Comment:
See https://github.com/JSPrismarine/JSPrismarine/blob/e50b40f01262d08a479f8073099d78a65f53c590/.babelrc

Reply:
(for typescript support in babel)